index.html

```diff
@@ -1,183 +1,228 @@
+          <h3 class="title is-3">
+            <a href="https://cowinmap.com">The Cowin Map</a>
+          </h3>
+          <p>#verified #covid resources from volunteers across #India on Google maps for quick, updated and easy access.
+          </p>
+          <p>Let’s help India win against covid!</p>
+        </span>
+
+        <span class="is-hidden-tablet">
+          <!-- Shown only on mobile -->
+          <h6 class="title is-5">
+            <a href="https://cowinmap.com">The Cowin Map</a>
+          </h6>
+          <p class="is-size-7">#verified #covid resources from volunteers across #India on Google maps for quick,
+            updated and easy access.</p>
+          <p class="is-size-7">Let’s help India win against covid!</p>
+        </span>
+      </div>
+    </div>
+  </header>
+
+  <div class="modal-wrap" id="modal-demo">
+    <div class="modal-container">
+      <div class="modal-header">
+        <h2 class="modal-title">Liability Notice</h2>
+      </div>
+      <div class="modal-content">
+        <p>The user understands that, The Cowin Map has made every attempt to ensure the accuracy and reliability of the
+          information provided through these links. However, the information is provided on an “as and when basis” and
+          “as
+          in” based on a fact check done by volunteers who are dedicated to help individuals and families in such
+          challenging times. The Cowin Map, however, do not accept any responsibility or liability for the accuracy,
+          content, completeness, legality or reliability of the information contained in any of these. The user
+          understands that no warranties, promises and/or representation of any kind, expressed or implied, are made by
+          The Cowin Map to the nature, standard, accuracy or otherwise of the information provided in these links nor to
+          the suitability or otherwise of the information to your particular circumstances. The user understands that,
+          The
+          Cowin Map shall not be liable for any loss or damage of whatsoever nature (direct, indirect, consequential, or
+          other) which may arise as a result of your use of or (inability to use) the information contained in any of
+          these links. The user understands that the information contained in these links, provides details like the
+          contact numbers etc of hospitals, agencies etc, the contents of which are not within the control of The Cowin
+          Map and The Cowin Map has no responsibility for the same. The user agrees to check and verify any/all
+          information relied upon basis in these links before acting upon the same. The user understands that The Cowin
+          Map hold no liability or responsibility for real time verification of the same.
+        </p>
+      </div>
+      <div class="modal-footer">
+        <button class='accept-btn'>I Understand</button>
+      </div>
+    </div>
+  </div>
+
+  <script>
```

Comment:
Let's move this Javascript to it's own Javascript file. This will keep it consistent with how we maintain all other Javascript files.